src/scikit_build_core/builder/builder.py

```diff
@@ -123,19 +187,29 @@ def configure(
         }
 
         # Add any extra CMake modules
-        eps = metadata.entry_points(group="cmake.module")
-        self.config.module_dirs.extend(resources.files(ep.load()) for ep in eps)
+        _handle_search_paths(
+            "cmake.module", self.settings.search.modules, self.config.module_dirs
+        )
```

Comment:
Should we unify the naming convention across these 3 to the entry-point name? I.e.:
```suggestion
        _handle_search_paths(
            "cmake.module", self.settings.search.module, self.config.module
        )
```